src/main/java/animals/Application.java

```diff
@@ -0,0 +1,47 @@
+package animals;
+
+import java.io.IOException;
+
+public final class Application extends TextInterface implements Runnable {
+    private final KnowledgeTree knowledgeTree;
+    private final StorageService storageService;
+    private final TreeServices treeServices;
```

Comment:
Why this class was named in plural?

Reply:
Because it provides several services(methods) that will be invoked if user will enter number of corresponding menu item. I'm sorry, but is such naming isn not conventional?

Reply:
Because it provides several services(methods) one of which will be invoked if user will enter number of corresponding menu item. I'm sorry, but is such naming isn not conventional?